docs/src/architecture/accounts.md

```diff
@@ -52,6 +52,27 @@ Interface for accounts. In Miden every account is a smart contract. It has an in
 
 *Note: Since code in Miden is expresed as MAST, every function is a commitment to the underlying code. The code cannot change unnoticed to the user because its hash would change. Behind any MAST root there can only be `256` functions*
 
+#### Example Account Code
+Currently, Miden provides two standard implementations for account code. 
+
+A basic user account exposes the following code as its interface.
+
+```
+    use.miden::contracts::wallets::basic->basic_wallet
+    use.miden::contracts::auth::basic
+
+    export.basic_wallet::receive_asset
+    export.basic_wallet::send_asset
+    export.basic::auth_tx_rpo_falcon512
+
+```
+
+That means, [note scripts](https://0xpolygonmiden.github.io/miden-base/architecture/notes.html#note-scripts) or transaction scripts can call those functions and in doing so, the account can receive and send assets and authenticate the transaction. It is important to know, that without correct authentication, a note cannot successfully invoke receive and send asset.
```

Comment:
It is unclear what "correct authentication" means.

Reply:
I changed the sentence to 

```
It is important to know, that without correct authentication, i.e. knowing the correct private Falcon key, a note cannot successfully invoke receive and send asset.
```

Reply:
is that better?

Reply:
Yeah, better. The only thing I would point out is that you said "Falcon key". 

For now it's the only auth scheme that we have but we could add some more in the future. 

You could change it to "private key" or something related to auth schemes.